.gitignore

```diff
@@ -1,3 +1,11 @@
 config.codekit
 .DS_Store
-/node_modules
\ No newline at end of file
+/node_modules
```

Comment:
As these changes are now out of sync with the progress from that tooling branch we want to just ignore them for this pull request. I think if you revert those commits on this branch so nothing actually changes then they will disappear from this PR

Reply:
So only have the changes relating to the stylus library in this PR